pubspec.yaml

```diff
@@ -11,6 +11,7 @@ environment:
   flutter: ">=2.5.0"
 
 dependencies:
+  collection: ^1.16.0
```

Comment:
where do we use this? 👀

Reply:
this is used for Chantal's change to use `firstWhereOrNull` in dvc_feature and dvc_variable